ocs_ci/ocs/resources/pod.py

```diff
@@ -985,3 +1034,17 @@ def get_operator_pods(operator_label=constants.OPERATOR_LABEL, namespace=None):
     operators = get_pods_having_label(operator_label, namespace)
     operator_pods = [Pod(**operator) for operator in operators]
     return operator_pods
+
+
+def upload(pod_name, localpath, remotepath):
+    """
+    Upload a file to pod
+
+    Args:
+        pod_name (str): Name of the pod
+        localpath (str): Local file to upload
+        remotepath (str): Target path on the pod
+
+    """
+    cmd = f"oc cp {os.path.expanduser(localpath)} {pod_name}:{remotepath}"
```

Comment:
We can later on add this option (No need for this PR):  --no-preserve=false: The copied file/directory's ownership and permissions will not be preserved in the container

Reply:
Agreed. I will take it up as followup PR